packages/privacy-toolset/src/do-not-sell-dialog/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormToggle } from '@wordpress/components';
+
+import './styles.scss';
+
+export type DoNotSellDialogProps = {
+	content: {
+		title: string;
+		longDescription: ( string | React.ReactNode )[];
```

Comment:
I am not _sure_ this is better, but I had a thought that this prop may better be served as a React child so the markup can just be written in HTML instead of passing an array of nodes or strings. I think it makes sense in this context for the bulk of the content to be passed as a Child instead of a nested prop.

This could give the user of this component more customizability of the content in the dialog if needed without the content automatically being put into `<p>` tags (though I know that's going to be what is in here possibly 100% of the time)

Reply:
It's a great idea, thanks!